services/workflows-service/src/business-report/business-report.controller.internal.ts

```diff
@@ -361,19 +357,23 @@ export class BusinessReportControllerInternal {
     }),
     RemoveTempFileInterceptor,
   )
-  async createBatchReport(
+  async createBusinessReportBatch(
     @UploadedFile() file: Express.Multer.File,
-    @Body() body: CreateBatchBusinessReportDto,
+    @Param() { type }: CreateBusinessReportBatchQueryParamsDto,
+    @Body() body: CreateBusinessReportBatchBodyDto,
```

Comment:
```
  @ApiConsumes('multipart/form-data')
  @UseInterceptors(
    FileInterceptor('file', {
      storage: getDiskStorage(),
      fileFilter,
    }),
    RemoveTempFileInterceptor,
```


For files we used to set interceptor using Multer and validate maximum payload size and you can also validate the mimetype